signalfx/data_source_pagerduty_integration.go

```diff
@@ -0,0 +1,21 @@
+package signalfx
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func dataSourcePagerDutyIntegration() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourcePagerDutyIntegrationRead,
+		Schema: map[string]*schema.Schema{
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+		},
+	}
+}
+
+func dataSourcePagerDutyIntegrationRead(d *schema.ResourceData, meta interface{}) error {
+	return nil
```

Comment:
The lookup here would be done by Integration Name. However the Go SDK does not have this function implemented yet.

Need to create it in the Go SDK and then port it.

Reply:
This has been ported